### ISSUE TITLE
Failed to initialize wrangler bindings proxy The Workers runtime failed to start

### ISSUE THREAD

**Issue by mavrick** [pi0/nitro-cloudflare-dev#34]
```
 ERROR  *** Received structured exception #0xc0000005: access violation; stack: 7ff9ed0e301f 7ff788f594eb 7ff788f595b3 7ff788f4d1ec 7ff788f4d197 7ff7887c832e 7ff7887c863f 7ff7876b1706 7ff7876b14ea 7ff78894047f 7ff788947636 7ff78894099c 7ff788947636 7ff78893e30c 7ff7876a1551 7ff789fdc8df 7ffa0e27257c 7ffa0eb4af27


 ERROR  Failed to initialize wrangler bindings proxy The Workers runtime failed to start. There is likely additional logging output above.

  at Miniflare2.#assembleAndUpdateConfig (node_modules\miniflare\src\index.ts:1343:10)
  at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  at Mutex.runWith (node_modules\miniflare\src\workers\shared\sync.ts:66:45)
  at Miniflare2.#waitForReady (node_modules\miniflare\src\index.ts:1425:3)
  at Miniflare2._getProxyClient (node_modules\miniflare\src\index.ts:1631:3)
  at Miniflare2.getBindings (node_modules\miniflare\src\index.ts:1654:23)
  at async getPlatformProxy (node_modules\wrangler\wrangler-dist\cli.js:210786:20)
  at _getPlatformProxy (node_modules\nitro-cloudflare-dev\dist\runtime\plugin.dev.mjs:41:17)
  ```
  
  https://github.com/cloudflare/workers-sdk/issues/4709
  
  This just started happening. 
  
  ```
  "dependencies": {
    "@nuxtjs/critters": "^0.7.1",
    "@nuxtjs/device": "^3.1.1",
    "@nuxtjs/google-fonts": "^3.2.0",
    "@nuxtjs/seo": "^2.0.0-rc.15",
    "@nuxtjs/tailwindcss": "^6.12.1",
    "@splidejs/vue-splide": "^0.6.12",
    "nuxt-gtag": "^2.0.7",
    "tailwindcss": "^3.4.6",
    "tailwindcss-textshadow": "^2.1.3",
    "video.js": "^8.16.1",
    "videojs-youtube": "^3.0.1",
    "vue": "^3.4.31",
    "vue-router": "^4.4.0",
    "vue-tippy": "^6.4.4",
    "vue3-mq": "^3.2.0"
  },
  "devDependencies": {
    "@cloudflare/workers-types": "^4.20240712.0",
    "nitro-cloudflare-dev": "^0.1.5",
    "nuxt": "^3.12.3",
    "wrangler": "^3.65.0"
  }
  ```

**Comment by Atinux** [comment]
Can you try with wrangler 3.64.0 ?

**Comment by pi0** [comment]
It should be fixed in [v0.1.6](https://github.com/pi0/nitro-cloudflare-dev/releases/tag/v0.1.6) (you can recreate your lock file).
